index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div>
+      <nav>
+        <h3>My logo</h3>
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </div>
```

Comment:
- [ ] According to the requirements, in place of `div` you are to use a `header` tag to house the logo and menu sections. You can refer to [this article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more explanation
- [ ] Kindly ensure that only the menu bar icon is inside the nav. The `nav` tag is for semantics reason and the right position for convention and user experience for mobile devices